api/requirements.txt

```diff
@@ -31,4 +31,4 @@ requests
 requests_oauthlib
 
 # deployment
-uwsgi
+# uwsgi
```

Comment:
I am not sure this should be part of this pull request

Reply:
Yes yes. just an error.